hubblestack/extmods/utils/customUserConf.py

```diff
@@ -0,0 +1,45 @@
+import json
+import yaml
+import logging
+import hubblestack.log
+
+log = logging.getLogger(__name__)
+basePath = '/etc/hubble/hubble.d/'
```

Comment:
Handle this path for both linux and windows based on platform

Reply:
Done